packages/libs/react-ui/src/components/Notification/Notification.tsx

```diff
@@ -0,0 +1,82 @@
+import { SystemIcon } from '../Icons';
+
+import {
+  cardTitleClass,
+  colorVariants,
+  contentClass,
+  expandClass,
+  footerClass,
+  headerContainerClass,
+  iconContainerClass,
+  iconContainerFullWidthClass,
+  simpleClass,
+  simpleContentClass,
+} from './Notification.css';
+
+import classNames from 'classnames';
+import React, { FC } from 'react';
+
+export interface INotificationProps {
+  icon?: (typeof SystemIcon)[keyof typeof SystemIcon];
+  title: string;
+  children?: React.ReactNode;
+  displayCloseButton?: boolean;
+  expand?: boolean;
+  color?: keyof typeof colorVariants;
+  simple?: boolean;
+
+  headerIcons?: React.ReactNode;
+  footer?: React.ReactNode;
```

Comment:
Based on our previous discussion regarding subcomponents, I believe these shouldn't be passed as props, but rather have Notification sub components that people can use to compose a notification.